assets/src/edit-story/components/panels/panel/panel.js

```diff
@@ -43,13 +45,39 @@ function Panel({
   initialHeight = null,
   ariaLabel = null,
   ariaHidden = false,
+  persistenceKey,
```

Comment:
Could we just use `name` for the key?

And then only use a prop for a boolean flag to toggle whether changes should persist?

Reply:
This will cause conflicts between the style and color preset panel no? since they both have the name `stylepreset` hardcoded?

Reply:
Hmm perhaps they should just pass 

```
name={`stylepreset-${presetType}`}
```

?

Reply:
maybe a prefixed `presetType` may be best for the key for now?

Reply:
@littlemilkstudio I think we just sent this at the same time? 😅